company/team/index.md

```diff
@@ -232,4 +232,11 @@ To add yourself to this page, copy the following template, paste it at the end o
 - [quin@sourcegraph.com](mailto:quin@sourcegraph.com), [quinwith1n](https://github.com/qwith1n)
 - Quin has always loved traveling and seeing the world. He lives in Phoenix with his wife and 2 french bulldogs.  Quin also enjoys most sports, especially ping pong, reading, and playing his guitar.  Although his French bulldogs do not enjoy it when he attempts to play the guitar.
 
+## Asdine El Hrychy
+
+- Software Engineer
+- Lyon, France 🇫🇷/ Dubai, UAE 🇦🇪
+- [asdine@sourcegraph.com](mailto:asdine@sourcegraph.com), [asdine](https://github.com/asdine)
+- Asdine loves experimenting and building bizarre software while enjoying an entire pot of coffee. Before Sourcegraph, he was writing tools and frameworks for his colleagues at Heetch. He also hates dogs, especially the small ones, essentially because they hate him.
```

Comment:
So you're a cat person? :D What about normal sized or bigger dogs? Look how cute they are.


![](https://media0.giphy.com/media/6Umkh0GwRYhfG/giphy.gif?cid=5a38a5a23e47d544aae9e815fe4f98cdced09ae843408d8e&rid=giphy.gif)

Reply:
Puppies are fine, I'm mostly talking about Chiuahas, Poodles and other creatures from hell